scripts/jenkins/Jenkinsfile.release

```diff
@@ -7,6 +7,9 @@ properties([
         string(name: 'TAG',
                defaultValue: 'v0.0.0',
                description: 'CLUO tag to release'),
+        booleanParam(name: "PUSH_LATEST",
```

Comment:
If you want to make a PR moving the example to use the latest published version I can delete that commit from this happily enough.